### DISCUSSION TITLE
Fountain Syntax / Formatting

### DISCUSSION THREAD

**Discussion by Reproog** [vkbo/novelWriter#2034] [Ideas]
Hi, 

I'm currently working on a small project where I need screenplay formatting in some chapters.

I know NovelWriter is not an app for writing screenplays per se. However, I could imagine that it would be "relatively" easy to add "[Fountain Syntax](https://fountain.io/) Highlight".

(For example, you can specify that the following document is a "Fountain" document. Then you could use the formatting there and the correct formatting could be displayed in the "View Document" tab (screenplay formatting) )

Would it be possible to implement something like this? Or what do you think of this idea?

**Comment by vkbo** [comment]
I'm considering adding a feature for pre-formatted text (aka code block), which could be quite suitable for this purpose too. Inside such a block you can add whatever you want without the formatting processing messing it up.

Fountain syntax support itself will not be added though. It would require quite a lot of modification to make it work. See the "Frequently Requested Features" pinned issue thread for more info: #815
